zshrc

```diff
@@ -45,12 +45,13 @@ unsetopt nomatch
 
 # vi mode
 bindkey -v
-bindkey "^F" vi-cmd-mode
-bindkey jj vi-cmd-mode
```

Comment:
As it stands, I think using `^J` here is a deal-breaker, as it clashes with vim-tmux-navigator and our vim pane navigation shortcuts. If there's another binding that doesn't conflict with core functionality I think it's worth discussing.